sktm/patchwork.py

```diff
@@ -871,7 +878,7 @@ def get_rpc(self, baseurl):
         Returns:
             The XML RPC interface for the Patchwork
         """
-        rpc = xmlrpclib.ServerProxy("%s/xmlrpc/" % baseurl)
+        rpc = xmlrpclib.ServerProxy(join_with_slash(baseurl, "xmlrpc/"))
```

Comment:
The returned value by the `join_with_slash` function will not be the same as the original one -- the slash at the end of the result is missing.

Reply:
Right. `join_with_slash` will remove the trailing slash